### PR TITLE
Add phone and bundle strings

### PR DESCRIPTION
**Pull request by Vinnl**
For those, Spanish and French translation is enough.

Note that some additional work might need to be done for Pontoon to migrate properly before merging this PR: https://github.com/mozilla-l10n/fx-private-relay-l10n/issues/104.

### CODE REVIEW

**Comment by flodolo** [comment]
@peiying2 
Just to confirm, have you synced with @mathjazz around this? I would strongly suggest to test it on stage, if that's not already planned, before merging for prod.

File: en/phones.ftl [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+
+phone-onboarding-step4-country = United States and Canada
+phone-onboarding-step4-body = These available phone number masks are similar to your true phone number.
+phone-onboarding-step4-sub-body = Once you register a phone number mask, you cannot change it.
+phone-onboarding-step4-smiliar-phone = Similar to { $phone_number }
+phone-onboarding-step4-input-search = Enter an area code
+phone-onboarding-step4-button-more-options = Show me other options
+phone-onboarding-step4-button-register-phone-number = Register phone number mask
+phone-onboarding-step4-search-results-body = Phone number masks available in <strong>{ $location }</strong>. Once you register a phone number mask, you cannot change it.
+phone-onboarding-step4-confirm-message = To confirm your your phone number mask, enter it below. This can’t be changed later.
```
Comment (peiying2) [inline en/phones.ftl:91]: duplicate `your`

File: en/phones.ftl [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+phone-dashboard-forwarding-toggle-enabled = Forwarding enabled
+phone-dashboard-forwarding-toggle-disabled = Forwarding disabled
+phone-dashboard-forwarding-enabled = { -brand-name-relay } is currently forwarding all phone calls and text messages to your true phone number.
+phone-dashboard-forwarding-blocked = { -brand-name-relay } is blocking all phone calls and text messages—you will not receive anything from your phone number mask.
+phone-dashboard-senders-header = Callers and text senders
+phone-dashboard-sender-table-title-sender = Sender
+phone-dashboard-sender-table-title-activity = Latest Activity
+phone-dashboard-sender-table-title-action = Action
+phone-dashboard-sender-disabled-body =  You have disabled the Caller and Sender log. Go to your settings to
+        enable { -brand-name-relay } to keep a log of your callers and senders.
```
Comment (peiying2) [inline en/phones.ftl:140]: Is there a reason this string is broken into two lines?

File: en/phones.ftl [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+    }
+
+phone-onboarding-step3-input-placeholder = Enter 6-digit code
+phone-onboarding-step3-button-cta = Confirm my phone number
+phone-onboarding-step3-button-edit = Edit true phone number
+phone-onboarding-step3-button-resend = Resend verification code
+phone-onboarding-step3-error-exipred = Try again—the time limit expired.
+phone-onboarding-step3-error-cta = Send a new code
+phone-onboarding-step3-code-fail-title = Wrong verification code
+phone-onboarding-step3-code-fail-body = Please try again or request a new code
```
Comment (peiying2) [inline en/phones.ftl:75]: add a period at the end of the string.

File: en/phones.ftl [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+
+phone-settings-caller-sms-log = Caller and texts log
+phone-settings-caller-sms-log-description = Allow { -brand-name-firefox-relay } to keep a log of your callers and text senders.
+phone-settings-caller-sms-log-warning = If you decide to opt out from this preference, you will lose the ability to block and reply to senders or callers. If you’ve blocked any, they will become unblocked and your existing caller and text sender log will be permanently cleared from your history.
+
+## Phone Resend SMS Banner
+
+phone-banner-resend-welcome-sms-cta = Resend welcome text
+phone-banner-resend-welcome-sms-title = Quick Tip
+phone-banner-resend-welcome-sms-body =  Remember to save the contact we shared with you by text to help you identify messages forwarded by { -brand-name-relay }. Can’t find it?
```
Comment (peiying2) [inline en/phones.ftl:113]: Is there anything after `Can't find it?`

**Review by peiying2 — COMMENTED** [review]
Do NOT merge before Pontoon migration of translated string is completed. 

In addition to the individual brand names, the entries are repeated for every file. If we were to add entries in the future, all on-production files will be added. They need to be individual submitted even though most don't require translation. Let's consider creating a brand file if it is not too much engineering work.

**Comment by Vinnl** [comment]
> the entries are repeated for every file

This is just referring to the brand strings, right? Do you have a link to documentation on how to do that, or alternatively an example repo that has done this? Happy to implement that if I can find out how to :)

> If we were to add tries in the future, all on-production files will be added. They need individual submitted even though most don't require translation.

I'm sorry, I don't understand this sentence. What does "tries" mean here?

File: en/phones.ftl [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+            *[other] Please enter the verification code that was sent to <span>{ $phone_number } </span> within <strong>{ $remaining_seconds } seconds</strong>.
+        }
+        *[other] Please enter the verification code that was sent to <span>{ $phone_number } </span> within <strong>{ $remaining_minutes }:{ NUMBER($remaining_seconds, minimumIntegerDigits: 2) } minutes</strong>.
+    }
+
+phone-onboarding-step3-input-placeholder = Enter 6-digit code
+phone-onboarding-step3-button-cta = Confirm my phone number
+phone-onboarding-step3-button-edit = Edit true phone number
+phone-onboarding-step3-button-resend = Resend verification code
+phone-onboarding-step3-error-exipred = Try again—the time limit expired.
```
[the remark this answers is not in the document]
Comment (Vinnl) [reply en/phones.ftl]: 2a33745

File: en/phones.ftl (hunk 1 above)
Comment (Vinnl) [reply en/phones.ftl]: Woah, not even QA caught that: 2a33745

File: en/phones.ftl (hunk 2 above)
Comment (Vinnl) [reply en/phones.ftl]: Don't think so: 2a33745

File: en/phones.ftl (hunk 3 above)
Comment (Vinnl) [reply en/phones.ftl]: 2a33745

File: en/phones.ftl (hunk 4 above)
Comment (Vinnl) [reply en/phones.ftl:114]: Yeah, but it's on a separate line it looks like:

image

I added a comment to clarify that in 2a33745, is that enough? I'm not sure if I have a good way of incorporating those into the same string.

**Comment by peiying2** [comment]
> I'm sorry, I don't understand this sentence. What does "tries" mean here?

Looks like you already did what I meant to suggest you to do. Sorry for the incoherent comment, lol - not sure what happened - can't blame the fat fingers. I meant to say the individual brand names (not comments), and "entries" instead of "tries". You already took care of it. Thank you!

**Comment by peiying2** [comment]
LGTM! Thanks for the quick update. May need a dev to check the section on the minute/second remaining to verify number. Want to make sure the logic is correct.

**Comment by Vinnl** [comment]
@peiying2 OK, and are you reaching out to that dev (is that mathjazz?)? Also, does flod's "testing on stage" mean a stage env for Pontoon, or just the regular Relay stage environment? We'll definitely do the latter, but I don't know how to do the former.

(I'm also adding a couple of commits with some more new strings.)

**Comment by peiying2** [comment]
> @peiying2 OK, and are you reaching out to that dev (is that mathjazz?)? Also, does flod's "testing on stage" mean a stage env for Pontoon, or just the regular Relay stage environment? We'll definitely do the latter, but I don't know how to do the former.
> 
> (I'm also adding a couple of commits with some more new strings.)

For testing of added files, the staging is on Pontoon to make sure the files show up only in the locales specified. 

For the code on the time (hr/sec), if your team has already reviewed it, and it looks correct, then I am good. On our side, it would be flod, but he is off this week.

**Comment by Vinnl** [comment]
On our side, those strings show up fine in English, so I think we're good. Does that mean that once https://github.com/mozilla-l10n/fx-private-relay-l10n/issues/104 is merged, there are no more blockers for this one to be merged too?

**Comment by mathjazz** [comment]
I have tested this patch locally to verify the migration plan outlined in #104 works as expected and it does. :)

I've noticed a few things though, which I think we should fix before we proceed:

1. Seems like the newly created `brands.ftl` file is not referenced in the TOML file, which means it will not be exposed to any locale. I guess that's not intentional?

1. For future reference, please note that moving source strings from one file to another (e.g. `brands.ftl`) also requires moving any existing translations corresponding to these strings. Otherwise we'll lose them. For this patch, I'll do the migration of translations in the DB.

1. As per convention, I'd also suggest we move the TOML file to the root of the repository and rename it to l10n.toml.

**Comment by Vinnl** [comment]
Whoops, thanks for the check @mathjazz, still have to get used to the TOML file :) Hopefully we shouldn't need to move strings from one file to another after this is merged.

I also renamed the TOML file as per your suggestion.

**Comment by Vinnl** [comment]
@peiying2 I updated a couple of strings for https://github.com/mozilla/fx-private-relay/pull/2649: https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/103/commits/a8b16dec29bccde1596db9b5c8c827f1ae4905fd

**Comment by peiying2** [comment]
> @peiying2 I updated a couple of strings for [mozilla/fx-private-relay#2649](https://github.com/mozilla/fx-private-relay/pull/2649): [a8b16de](https://github.com/mozilla-l10n/fx-private-relay-l10n/commit/a8b16dec29bccde1596db9b5c8c827f1ae4905fd)

The added strings look good.

File: l10n.toml [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+  "ja",
+  "nl",
+  "pt-BR",
+  "ru",
+  "sk",
+  "skr",
+  "sv-SE",
+  "uk",
+  "zh-CN",
+  "zh-TW"
```
[the remark this answers is not in the document]
Comment (mathjazz) [reply l10n.toml:30]: It's a good practice, but it's not required.

**Review by mathjazz — APPROVED** [review]
I tested the latest changes and it worked as expected locally.

I'll merge now and do the migration in Pontoon.

**Pull request merged**